### ISSUE TITLE
Installation just hangs on Mac

### ISSUE THREAD

**Issue by souljacker** [docker/for-mac#2423]
### Expected behavior

Running Docker for the first time should install the app

### Actual behavior

The app hangs at "initialising docker", whenever I click it, this error message is displayed

<img width="579" alt="screen shot 2018-01-13 at 17 39 28" src="https://user-images.githubusercontent.com/8139090/34909485-3e1fbcac-f889-11e7-9f6d-755ed802b59b.png">

### Information

  - Here's my diagnosis id 
D205F1DB-BAF4-4EBD-AA56-4CEB8B374231

And here's the actual dianogsis
```
Docker for Mac: version: 17.12.0-ce-mac47 (72b93a017350990850ddc37cd341bd16fce3e911)
macOS: version 10.13.1 (build: 17B1003)
logs: /tmp/D205F1DB-BAF4-4EBD-AA56-4CEB8B374231/20180113-174807.tar.gz
failure: com.docker.vpnkit is not running
[OK]     db.git
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/paranhos/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/paranhos/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/paranhos/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         slirp check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/paranhos/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[OK]     disk
```

I tried uninstalling docker  by using this bash script (docker actual menu is non responsive). I also tried  reseting to factory defaults, but nothing happens.

Here's my Mac info

<img width="268" alt="screen shot 2018-01-13 at 17 53 20" src="https://user-images.githubusercontent.com/8139090/34909555-add18912-f88a-11e7-8e9d-61b15359e458.png">

**Comment by souljacker** [comment]
BTW, I can run docker from bash

**Comment by NadahF** [comment]
I am having the same issue, have you figured anything out?

**Comment by souljacker** [comment]
@NadahF nope

It seems there are a lot of similar issues here. None with a solution :(

**Comment by nuke99** [comment]
still the same issue. any helps?

**Comment by akimd** [comment]
Hi,

This is the first time Docker was installed on this machine?  Did you install it in the Applications folder and run it from there?

**Comment by souljacker** [comment]
@akimd it was not the first time, no. I thought it might be related to leftovers  from previous installations, but I tried cleaning it all and still got the same issues.

> Did you install it in the Applications folder and run it from there?

Yes

**Comment by akimd** [comment]
Could you have a look at https://github.com/docker/for-mac/issues/2383#issuecomment-358246040 and tell me if you have the same behaviour, and if the script does solve your problem?

**Comment by gtardif** [comment]
If you get a chance, can you uninstall and then try this test build: https://download-stage.docker.com/mac/pr/21935/Docker.dmg. 
Keep in mind this is a test build. Ignore any update it might propose, and after the test you will have to reinstall an Edge or Stable version to get proper updates again. 
Let us know if this works (or upload diagnostics if not) ; if this works fine we'll integrate this fix in a proper release soon.

**Comment by akimd** [comment]
Could you please check if the following image fixes your problems? You might need to reset to factory defaults...

https://download-stage.docker.com/mac/pr/21949/Docker.dmg

This is a test build. Ignore any update it might propose, and after testing, reinstall an Edge or Stable version to get proper updates again. Thanks!

**Comment by gtardif** [comment]
We just released 17.12.0-ce-mac49 on the Stable channel that contains the fix mentioned above. (and 18.01 on the Edge channel should also fix this)

**Comment by souljacker** [comment]
18.01 edge worked for me
